whois-nrtm4/src/main/java/net/ripe/db/nrtm4/NotificationFileGenerationService.java

```diff
@@ -0,0 +1,43 @@
+package net.ripe.db.nrtm4;
+
+import net.ripe.db.nrtm4.persist.NrtmDocumentType;
+import net.ripe.db.nrtm4.persist.NrtmSource;
+import net.ripe.db.nrtm4.persist.NrtmVersionInformationDao;
+import net.ripe.db.nrtm4.persist.VersionInformation;
+import net.ripe.db.nrtm4.publish.SnapshotFile;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+
+@Service
+public class NotificationFileGenerationService {
+
+    private final NrtmVersionInformationDao versionDao;
+
+    public NotificationFileGenerationService(
+            final NrtmVersionInformationDao nrtmVersionInformationDao
+    ) {
+        versionDao = nrtmVersionInformationDao;
+    }
```

Comment:
this.versionDao? Also rename the variable to nrtmVersionDao ?

Reply:
done in 'transform-rpsl-to-nrtm-json'